src/Discovery/Discovery.css

```diff
@@ -343,6 +344,12 @@
   border-color: var(--g3-color__base-blue);
 }
 
+.discovery-action-bar-button:hover {
+  color: var(--secondary-color_light);
+  border-color: var(--secondary-color_light);
```

Comment:
This isn't ideal. I'd prefer #fff, but then the text becomes invisible on BRH because that's the primary color

Reply:
I don't think BRH has a primary color, the buttons in BRH has the style that defined within their component library

I can see the point, but since HEAL's color scheme is highly customized, I would suggest to put this entrie `.discovery-action-bar-button:hover` section out into `gitops.css` rather than put them in here are default styles

Reply:
or we can set `#fff` as default color for `color` and `boarder-color`, and `--g3-color__base-blue` for `background-color`

then overrider the `background-color` in HEAL's gitops.css